content/docs/components-and-props.md

```diff
@@ -154,11 +153,11 @@ function Comment(props) {
 
 [](codepen://components-and-props/extracting-components)
 
-It accepts `author` (an object), `text` (a string), and `date` (a date) as props, and describes a comment on a social media website.
+यह `author` (एक ऑब्जेक्ट), `text` (एक string), और `date` (एक date) को props के जैसे स्वीकार करता है|
```

Comment:
and describes a comment on a social media website has been removed while translating

Reply:
props के जैसे -> props ke dwaara